bom/pom.xml

```diff
@@ -820,6 +820,11 @@
                 <artifactId>s3</artifactId>
                 <version>${awsjavasdk.version}</version>
             </dependency>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>s3-control</artifactId>
```

Comment:
Did we verify this matches the one generated by the release automation?

Reply:
Nope, will verify.

Reply:
Appears that we remove spaces rather than replace with a dash looking at other services. Updated.